src/k8s/requests/v2v/utils.js

```diff
@@ -1,9 +1,30 @@
+import { alphanumericRegex } from '../../../utils';
+
 /**
  * Based on V2V Provider Pod manifest.yaml
  */
 
 const MAX_LEN = 30;
 
+// TODO: this needs to be improved to conform validations.js::validateDNS1123SubdomainValue()
+const alignWithDNS1123 = str => {
+  if (!str) {
+    return '';
+  }
+
+  // starts with alphaNum
+  if (!str.charAt(0).match(alphanumericRegex)) {
+    return alignWithDNS1123(str.slice(1));
```

Comment:
this can be done much easier without a recursion

```ts
str.replace(/^[^a-zA-Z0-9]*/, '').replace(/[^a-zA-Z0-9]*$/, '')
```

Reply:
sure, but I like recursion :-)

I can change it.